coc-settings.json

```diff
@@ -23,12 +25,19 @@
         "dart": {
             "command": "dart",
             "args": [
-                "/usr/lib/dart/bin/snapshots/analysis_server.dart.snapshot",
+                "/usr/local/opt/dart/libexec/bin/snapshots/analysis_server.dart.snapshot",
                 "--lsp"
             ],
             "filetypes": ["dart"],
             "disableDynamicRegister": true,
             "trace.server": "verbose"
+        },
+        "terraform": {
+            "command": "terraform-lsp",
```

Comment:
Maybe I could remove it I work with terraform that is why it is there but it is really specif and also we need to install the terraform-lsp so I am not sure